base/applications/osk/main.c

```diff
@@ -212,7 +212,7 @@ int OSK_DlgInitDialog(HWND hDlg)
     Globals.hBrushGreenLed = CreateSolidBrush(RGB(0, 255, 0));
 
     /* Set a timer for periodics tasks */
-    Globals.iTimer = SetTimer(hDlg, 0, 200, NULL);
+    Globals.iTimer = SetTimer(hDlg, 0, 1, NULL);
```

Comment:
Maybe use `USER_TIMER_MINIMUM` directly for `uElapse`, since it gets set to it anyway, because `1 < USER_TIMER_MINIMUM`?

Reply:
IIUC this timer unconditionally calls InvalidateRect three times. That sounds like a terrible thing to do every 200ms, let alone every 10ms. Can we fix this with something that won't eat all your CPU and cause a ton of flickering?